StaticAnalyzer/views/android/dvm_permissions.py

```diff
@@ -163,6 +163,61 @@
         "C2D_MESSAGE": ["signature", "Allows cloud to device messaging", "Allows the application to receive push notifications."],
 
         "ADD_VOICEMAIL": ["dangerous", "add voicemails into the system", "Allows an application to add voicemails into the system."],
+
+        "ACCEPT_HANDOVER": ["dangerous", "", "Allows a calling app to continue a call which was started in another app.  An example is a video calling app that wants to continue a voice call on the user's mobile network."],
+        "ACCESS_NOTIFICATION_POLICY": ["normal", "", "Marker permission for applications that wish to access notification policy."],
+        "ANSWER_PHONE_CALLS": ["dangerous", "", "Allows the app to answer an incoming phone call."],
+        "BIND_ACCESSIBILITY_SERVICE": ["signature", "", "Must be required by an"],
+        "BIND_AUTOFILL_SERVICE": ["signature", "", "Must be required by a"],
+        "BIND_CARRIER_MESSAGING_SERVICE": ["normal", "", ""],
+        "BIND_CARRIER_SERVICES": ["signature", "", "The system process that is allowed to bind to services in carrier apps will have this permission. Carrier apps should use this permission to protect their services that only the system is allowed to bind to."],
+        "BIND_CHOOSER_TARGET_SERVICE": ["signature", "", "Must be required by a"],
+        "BIND_CONDITION_PROVIDER_SERVICE": ["signature", "", "Must be required by a"],
+        "BIND_DREAM_SERVICE": ["signature", "", "Must be required by an"],
+        "BIND_INCALL_SERVICE": ["signature", "", "Must be required by a"],
+        "BIND_MIDI_DEVICE_SERVICE": ["signature", "", "Must be required by an"],
+        "BIND_NFC_SERVICE": ["signature", "", "Must be required by a"],
+        "BIND_NOTIFICATION_LISTENER_SERVICE": ["signature", "", "Must be required by an"],
+        "BIND_PRINT_SERVICE": ["signature", "", "Must be required by a"],
+        "BIND_QUICK_SETTINGS_TILE": ["normal", "", "Allows an application to bind to third party quick settings tiles."],
+        "BIND_REMOTEVIEWS": ["normal", "", "Must be required by a"],

```

Comment:
I see a lot of `Must be required by a` , `Must be required by an` 
Is it due to a bug in scraping script?
Could you please look into it?

Reply:
yes I see this is a bug when description contain a ref to another html section

Reply:
I have updated all the lines I see with missing info